integrations/build_system/xmake.rst

```diff
@@ -0,0 +1,69 @@
+.. xmake:
+
+XMake
+=========
+
+
+Install third-party packages:
+-------
+
+After version 2.2.5, xmake supports support for dependency libraries in third-party package managers.
+
+.. code-block:: lua
+   :caption: xmake.lua
+    
+    -- xmake.lua
+    
+    add_requires("CONAN::zlib/1.2.11@conan/stable", {alias = "zlib", debug = true})
```

Comment:
Please try to use the packages created from conan-center-index repo, not the legacy ones (packages from conan-center-indx don't have user/channel, and are all lowercase)